action.yml

```diff
@@ -12,6 +12,9 @@ inputs:
   tag-name:
     description: 'The tag associated with the release that contains the asset to download'
```

Comment:
Since you've made a change to the script I think the description here and on the readme should be updated to indicate it can handle `latest` as a tag name.